.claude/scripts/cluster-provision.sh

```
@@ -0,0 +1,376 @@
+
+    if [[ "$http_code" == "401" ]]; then
+        die "Authentication failed (HTTP 401). Your GANGWAY_TOKEN may be expired. Re-run: oc login ... && export GANGWAY_TOKEN=\$(oc whoami -t)"
+    fi
+    if [[ "$http_code" -lt 200 || "$http_code" -ge 300 ]]; then
+        die "Gangway API GET ${endpoint} failed (HTTP ${http_code}): ${body}"
+    fi
+    echo "$body"
+}
+
+save_state() {
+    mkdir -p -m 700 "$STATE_DIR"
```

Shellcheck (0.11.0)

[warning] When used with -p, -m only applies to the deepest directory.

(SC2174)